app/components/topic-feature/component.js

```diff
@@ -1,10 +1,11 @@
-import Ember from 'ember';
+import { inject as service } from '@ember/service';
+import Component from '@ember/component';
 
-export default Ember.Component.extend({
-  featureService: Ember.inject.service(),
-  feature: [],
+export default Component.extend({
+  featureService: service(),
 
-  didInsertElement () {
-    this._super.apply(this, arguments);
+  init () {
+    this._super(...arguments);
+    this.set('feature', []);
```

Comment:
Why is `this.set('feature', []);` preferable to `feature: [],`

Reply:
I had to make that change to get lint to pass, see https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/avoid-leaking-state-in-components.md